galaxy.yml

```diff
@@ -1,6 +1,6 @@
 namespace: community
 name: postgresql
-version: 1.0.0
+version: 1.0.1
```

Comment:
Once ansible-test has proper version_added tests (like ansible/ansible has), it's better to bump it to the next minor release 1.1.0. If you then decide that a patch release is needed reduce the version number before releasing it (to 1.0.1), release it, and re-bump to 1.1.0.

But as long as it's a WIP PR, no need to worry :)

Reply:
according to https://github.com/ansible/community/wiki/ReleasingCollections, we should adjust this to the next expected version. So, should we adjust the manual correspondingly?

Reply:
It depends on whether you want to use the version_added tests in the future, or not. If you want to use them, the tests somehow need to know which version they need to ensure in version_added. For that, you need the next minor/major release in galaxy.yml. But if you don't want to use that test (or only change it once that test actually is available), no need to change now.

Reply:
Ah, ok, thanks for the explaining, now got it. anyway, better to clarify it in the release guide

Reply:
i'll leave it as it is now, then will see

Reply:
put the initial back